src/orchard/misc.clj

```diff
@@ -1,6 +1,12 @@
 (ns orchard.misc
   (:require [clojure.string :as str]))
 
+(def ^:const windows-prefix
+  "Windows")
+
+(def ^:const os-windows?
```

Comment:
I'm not a big fan of this approach as well - why not make this a function? I doubt the performance impact would be meaningful.

Reply:
can turn into an fn, noprobs. but it won't ever change obviously, or? ;)

Reply:
My main issue with those constants is that they are hard to test. But it's not a big deal for me.

Reply:
indeed. changed it to fn and added a test